DirectProgramming/DPC++/N-BodyMethods/Nbody/src/GSimulation.cpp

```diff
@@ -0,0 +1,234 @@
+//==============================================================
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+// =============================================================
+// Copyright (c) 2019 Fabio Baruffa
+// Source: https://github.com/fbaru-dev/particle-sim
+// MIT License
+// =============================================================
+
+#include "GSimulation.hpp"
+#include "dpc_common.hpp"
```

Comment:
Please add the comments below for for the include of dpc_common.hp. it should look like this:

// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
#include "dpc_common.hpp"

as seen in https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot/src/main.cpp

Reply:
The comment is added in the new version.